docs/index.html

```diff
@@ -82,14 +89,14 @@
             <p>MemeInvestor is a r/MemeEconomy reddit bot that can help you invest in memes and make a fortune out of
                 it! </p>
             <!-- <p><a class="waves-effect waves-light btn orange darken-4" onclick="createWalletToast()"><i class="material-icons left">add</i>create your wallet</a> -->
-            <a href="#info1" class="waves-effect waves-light btn amber darken-1"><i
+            <a href="#info1" class="waves-effect waves-light btn yellow darken-3"><i
                     class="material-icons left">show_chart</i>learn more</a>
             </p>
             <div class="divider"></div>
             <div class="section">
                 <div class="row">
                     <div class="col s12 m4 l3">
-                        <h4 class="row-title">Overview</h4>
+                        <h4 class="no-margin-top">Overview</h4>
                         <p>Market active since 2018-06-29</p>
                         <p class="overview-text">Active investments: <span id="active-investments">--</span>, detaining
```

Comment:
Let's clean up the summary text to three lines:

"Active investments: ---"
"Total invested: --- M¢"
"Total balances: --- M¢"